File: SuperiorPackGroup/BLL/VendorBLL.vb
Imports DXDAL.SPGData
Imports DevExpress.Xpo

Public Class VendorBLL

    Public Shared Function VendorInUse(ByVal vendorID As Integer) As Boolean

        Try
            Dim cv As Vendor = Session.DefaultSession.GetObjectByKey(Of Vendor)(vendorID)
            If cv.Receivings.Count <> 0 Then
                Return True
            End If
        Catch ex As Exception
            Return False
        End Try

        Return False

    End Function

End Class

File: SuperiorPackGroup/BLL/VendorBLL.cs
//INSTANT C# NOTE: Formerly VB project-level imports:
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;

using DXDAL.SPGData;
using DevExpress.Xpo;

namespace SuperiorPackGroup
{
	public class VendorBLL
	{

		public static bool VendorInUse(int vendorID)
		{

			try
			{
				Vendor cv = Session.DefaultSession.GetObjectByKey<Vendor>(vendorID);
				if (cv.Receivings.Count != 0)
				{
					return true;
				}
			}
			catch (Exception ex)
			{
				return false;
			}

			return false;

		}

	}

}